src/controllers/tasks/active.pscs.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Templates } from "../../types/template.paths";
+import { Request, Response } from "express";
+import { TASK_LIST_PATH, urlParams } from "../../types/page.urls";
+
+export const get = (req: Request, res: Response) => {
+  const companyNumber = req.params[urlParams.PARAM_COMPANY_NUMBER];
+  const backLinkUrl = TASK_LIST_PATH.replace(`:${urlParams.PARAM_COMPANY_NUMBER}`, companyNumber);
```

Comment:
you can use the new function to build the url - https://github.com/companieshouse/confirmation-statement-web/blob/main/src/utils/url.ts#L3

Reply:
Updated this with a merge from main.